lang/en.yml

```
@@ -111,6 +111,8 @@ en:
     NEXT: 'Go to next record'
     PREVIOUS: 'Go to previous record'
     ViewPermissionsFailure: 'It seems you don''t have the necessary permissions to view "{ObjectTitle}"'
+    SAVETOASTMESSAGE: 'Saved {name} {title} successfully'
```

```suggestion
    SAVETOASTMESSAGE: 'Saved {type} {title} successfully'
```
Just to make it clearer what that's used for - it's not the name of the record, it's the type of record. Please also make this change to all other new strings.

---

Updated